### ISSUE TITLE
supportedInterfaces.VideoPlayer returns false for Echo Show

### ISSUE THREAD

**Issue by tflack** [alexa/alexa-skills-kit-sdk-for-nodejs#175]
Evaluating this.event.context.System.device.supportedInterfaces.VideoPlayer returns false on Echo Show (as well as non display devices).

this.event.context.System.device.supportedInterfaces.Display returns true on Echo Show.

The expectation is that this.event.context.System.device.supportedInterfaces.VideoPlayer should return true on Echo Show

**Comment by bclement-amazon** [comment]
Hey tflack,

Thanks for calling this out! This looks like an error in the README, which i'm in the process of updating.  The expectation should actually be if this.event.context.System.device.supportedInterfaces.VideoApp  is true

A device object sent from an Echo show today should look something like:
```javascript
"device": {
   "deviceId": "<deviceId>",
    "supportedInterfaces": {
          "AudioPlayer": {},
          "Display": {
                   "templateVersion": "1.0",
                    "markupVersion": "1.0"
           },
           "VideoApp": {}
      }
},
```

**Comment by bclement-amazon** [comment]
resolved with  commit 3882cc3cb99c5b00437bf6bf5a466602320b35bd
